pkg/aggregator/generated/applyconfiguration/internal/internal.go

```diff
@@ -5,8 +5,8 @@
 package internal
 
 import (
-	"fmt"
-	"sync"
+	fmt "fmt"
+	sync "sync"
```

Comment:
is this really needed?

Reply:
Oh wait this is a generated code. So I think this is ok.

Reply:
Right -- looks like they changed their codegen strategy